Gem/Code/Source/Automation/ScriptReporter.cpp

```diff
@@ -1081,4 +1091,74 @@ namespace AtomSampleViewer
         }
 
     }
+
+    void ScriptReporter::ExportTestResults()
+    {
+        uint32_t totalAssertCount = 0;
+        uint32_t totalErrorCount = 0;
+        uint32_t totalWarningCount = 0;
+        uint32_t totalScreenshotErrorCount = 0;
+        uint32_t totalScreenshotWarningCount = 0;
+        
+        for (const ScriptReport& scriptReport : m_scriptReports)
```

Comment:
If I'm reading this right, won't each script report write into `TestResults/exportedTestResults.txt` just stomping over the previous result? Could you describe the intention if you would?

Reply:
it should post each result that has a corresponding `ScriptReport` inside the collection of `ScriptReport` in `m_scriptReports` so the for loop iterates over each report inside `m_scriptReports` as a `ScriptReport` object, then assigns all data for THAT report. it should repeat and do it for each of them and in my testing it doesn't look like it's stomping the old data - but i am going to retest it right now to make sure

Reply:
i see what you mean now, it should write a new report file each time now